packages/TermsAndConditions/TermsAndConditions.jsx

```diff
@@ -0,0 +1,213 @@
+import React, { useState, useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import HairlineDivider from '@tds/core-hairline-divider'
+import DimpleDivider from '@tds/core-dimple-divider'
+import FlexGrid from '@tds/core-flex-grid'
+import Heading from '@tds/core-heading'
+import Box from '@tds/core-box'
+import { colorWhite, colorTelusPurple } from '@tds/core-colours'
+import { FadeAndReveal, Translate } from '@tds/shared-animation'
+
+import safeRest from '../../shared/utils/safeRest'
+import Clickable from '../../shared/components/Clickable/Clickable'
+
+import List from './List/List'
+
+const defaultCopy = {
+  en: {
+    headingClosed: 'View terms and conditions',
+    headingOpened: 'Hide terms and conditions',
+  },
+  fr: {
+    headingClosed: 'Voir les modalités et conditions',
+    headingOpened: 'Masquer les modalités et conditions',
+  },
+}
+
+const getCopy = copy => (typeof copy === 'string' ? defaultCopy[copy] : copy)
+
+const StyledClickable = styled(Clickable)({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'center',
+  backgroundColor: colorWhite,
+})
+
+const StyledExpandCollapseHeading = styled(Box)({
+  alignItems: 'center',
+})
+
+const StyledChevronContainer = styled.span({
+  position: 'relative',
+  width: '24px',
+  height: '24px',
+})
+
+const StyledChevron = styled.span(({ isOpen }) => ({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  transition: 'transform 300ms',
+  transform: `translate(6px, ${isOpen ? '-2px' : '-1px'})`,
+  [`${StyledClickable}:hover &`]: {
+    transform: !isOpen ? 'translate(6px, 2px)' : 'translate(6px, -5px)',
+  },
+}))
+
+const Chevron = ({ isOpen }) => (
+  <StyledChevron isOpen={isOpen}>
+    <svg
```

Comment:
Splitting SVGs off into their own files may make things more organized and allow for easier reusability later down the road.